src/main/java/org/jabref/gui/importer/NewDatabaseAction.java

```diff
@@ -4,24 +4,56 @@
 import org.jabref.gui.actions.SimpleCommand;
 import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.preferences.PreferencesService;
 
 /**
  * Create a new, empty, database.
  */
 public class NewDatabaseAction extends SimpleCommand {
 
     private final JabRefFrame jabRefFrame;
+    private final PreferencesService preferencesService;
+
     private final BibDatabaseMode mode;
+    private final boolean shouldUseDefaultMode;
 
-    public NewDatabaseAction(JabRefFrame jabRefFrame, BibDatabaseMode mode) {
+    /**
+     * Constructs a command to create a new library of a specific type
+     *
+     * @param jabRefFrame the application frame of JabRef
+     * @param preferencesService the preferencesService of JabRef
+     * @param mode the mode the new library is created in (BibTeX or biblatex)
+     */
+    public NewDatabaseAction(JabRefFrame jabRefFrame, PreferencesService preferencesService, BibDatabaseMode mode) {
```

Comment:
Is this constructor used somewhere? (I couldn't find it in the changelog, but maybe I'm missing something).
If not then, the `mode` and `shouldUseDefaultMode` variables can be deleted as well.

Reply:
No, I just thought I could leave it in, if someone decides to create a specific type of a library. Just a thought...

Reply:
I'm not a big fan of unused/dead code...

Reply:
meh... :wink: